common/views/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+import { classNames } from '../../../utils/classnames';
+import Space from '../styled/Space';
+
+const CheckboxLabel = styled.label.attrs({
+  className: classNames({
+    'flex flex--v-center': true,
+  }),
+})``;
+
+const CheckboxBox = styled.span.attrs({
+  className: classNames({
+    CheckboxBox: true,
```

Comment:
Why are we using this class name? Not `checkbox-box`?

Reply:
Yeah, this felt a bit weird and I thought alarm bells might ring in the PR comments. Should we agree to namespace from the component filename and use dashes?

Reply:
Ah, I see, this is to say if a Checkbox is in a CheckboxBox (Have we use container elsewhere instead of Box?) then do these things.

Wouldn't it make more sense for the parent to control the child style? Similar to the way we were thinking for Layouts, the child being ignorant of what context it might be in?

You also have this:
https://www.styled-components.com/docs/advanced#referring-to-other-components

Reply:
Ah yes! Using the `referring to other components` API, the problem goes away. Thanks.